lib/reference.js

```diff
@@ -141,7 +141,7 @@ Reference.updateTerminal = function (
     signatureToUse = signature;
   }
 
-  return promiseChain
+  return promiseChain // test
```

Comment:
This extra comment seems to have slipped in somehow.

Reply:
Oops! Thanks!